tests/test_appliancesmanager.py

```diff
@@ -120,29 +82,20 @@ async def test_fetch_appliances_returns_owned_and_shared_appliances(
 
 async def test_fetch_appliances_calls_owned_and_shared_methods(
     appliances_manager: AppliancesManager,
+    backend_selector_mock: BackendSelector,
+    aioresponses_mock,
 ):
-    shared_appliance_mock = get_mock_coro(None)
-    owned_appliance_mock = get_mock_coro(None)
-
-    appliances_manager._get_shared_appliances = shared_appliance_mock
-    appliances_manager._get_owned_appliances = owned_appliance_mock
+    owned_headers = appliances_manager._create_headers()
+    shared_headers = appliances_manager._create_headers(include_wp_brand_name=True)
```

Comment:
same as above

Reply:
if we use hardcoded headers in this test, then do we need the `test_get_owned_appliances_call_does_not_contain_wp_client_brand_header` and `test_get_shared_appliances_call_contains_wp_client_brand_header` tests? maybe we can remove them

Reply:
Yep, good call, I removed those after hardcoding the headers